### DISCUSSION TITLE
What is an undo shortcut for iPad?

### DISCUSSION THREAD

**Discussion by zhaob1n** [coder/code-server#4978] [Q&A] [answered]
I tried cmd + z but it doesn’t work. What is the correct shortcut?

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Firstly, have you modified the keybindings? If not, try checking for enabling them. Even if then it fails try checking iPadOS to forward keybindings.

**Reply by zhaob1n** [reply]
actually I just tried again and it works, strange enough
